packages/types/src/interfaces/sylo/v0/index.ts

```diff
@@ -0,0 +1,129 @@
+// Copyright 2019 Centrality Investments Limited
```

Comment:
Re-Introduced the old Sylo types we had when we supported v36 to have SYLO extrinsics to decode fine

Reply:
Which Sylo types have changed?

Reply:
I can see all the types(sylo) in the new API but they are auto generated from the definition and we don't have any test(Sylo).. I am investigating if the types we have currently are good for Sylo usage.. Trying to see how struct decodes...

Reply:
I have checked polkadot-api/packages/types/src/interfaces, there is no way to set the last parameter in a struct.. i.e
constructor (registry: Registry, Types: S, value: V | Map<unknown, unknown> | unknown[] | string = {} as V, **jsonMap**: Map<keyof S, string> = new Map()) {

In Sylo types, for example

```
const GROUP_JSON_MAP = new Map([['groupId', 'group_id']]);

export class Group extends Struct {
  constructor(registry: Registry, value) {
    super(
      registry,
      {
        groupId: 'H256',
        members: Vec.with(Member),
        invites: Vec.with(PendingInvite),
        meta: Meta,
      },
      value,
      GROUP_JSON_MAP
    );
  }
}
```

there is no way to set json_map in definition..